Zend/zend_vm_def.h

```diff
@@ -9544,6 +9544,136 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(204, ZEND_FRAMELESS_ICALL_0, UNUSED, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+	zend_frameless_function_0 function = (zend_frameless_function_0)ZEND_FLF_HANDLER(opline);
+	ZVAL_NULL(EX_VAR(opline->result.var));
+	function(EX_VAR(opline->result.var));
+	if (EG(exception)) {
+		HANDLE_EXCEPTION();
+	}
+	ZEND_VM_NEXT_OPCODE();
```

Comment:
Why don't you use ``ZEND_VM_NEXT_OPCODE_CHECK_EXCEPTION()``?

Reply:
The branch previously contained an assignment, thanks for noticing.